solutions/task_2.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+TASK: Create 'Fake' class to run code below without errors.
+"""
+
+import random
+import string
+
+
+class Fake(dict):
```

Comment:
dict? Why do you need it?

Reply:
![](https://www.asktheheadhunter.com/wp-content/uploads/2014/06/sucks-so-bad-300x145.jpg)